var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Gptune(CMakePackage):
+    """GPTune is an autotuning framework that relies on multitask and transfer
+    learnings to help solve the underlying black-box optimization problem using
+    Bayesian optimization methodologies."""
+
+    homepage = "https://gptune.lbl.gov/"
+    git      = "https://github.com/gptune/GPTune.git"
+    maintainers = ['liuyangzhuan']
+
+    version('master', branch='master')
+
+    variant('app', default=False, description='Build all HPC application examples')
+
+    depends_on('mpi', type=('build', 'link', 'run'))
+    depends_on('cmake@3.3:', type='build')
+    depends_on('jq', type='run')
+    depends_on('blas', type=('build', 'run'))
+    depends_on('lapack', type=('build', 'run'))
+    depends_on('scalapack', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-ipyparallel', type=('build', 'run'))
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-pandas', type=('build', 'run'))
+    depends_on('py-joblib', type=('build', 'run'))
+    depends_on('py-scikit-learn', type=('build', 'run'))
+    depends_on('py-matplotlib', type=('build', 'run'))
+    depends_on('py-pyyaml', type=('build', 'run'))
+    depends_on('py-scikit-optimize@master+gptune', type=('build', 'run'))
+    depends_on('py-gpy', type=('build', 'run'))
+    depends_on('py-lhsmdu', type=('build', 'run'))
+    depends_on('py-hpbandster', type=('build', 'run'))
+    depends_on('py-opentuner', type=('build', 'run'))
+    depends_on('py-ytopt-autotune@1.1.0', type=('build', 'run'))
+    depends_on('py-filelock', type=('build', 'run'))
+    depends_on('py-requests', type=('build', 'run'))
+    depends_on('py-cython', type=('build', 'run'))
+    depends_on('py-pyaml', type=('build', 'run'))
+    depends_on('py-mpi4py@3.0.3:', type=('build', 'run'))
+    depends_on('pygmo', type=('build', 'run'))
+    depends_on('openturns', type=('build', 'run'))
+
+    depends_on('superlu-dist@develop', when='+app', type=('build', 'run'))
+
+    conflicts('openmpi@:3')
+
+    def cmake_args(self):
+        spec = self.spec
+        fc_flags = []
+        if '%gcc@10:' in spec:
+            fc_flags.append('-fallow-argument-mismatch')
+        if self.spec.satisfies('%apple-clang@11:'):
```

Comment:
Could combine these into a single if statement using "or"

Reply:
will change this.